libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -273,6 +274,22 @@ const decorateColumns = async ({ content, separatorTagName = 'H5' } = {}) => {
   }
 };
 
+const decorateCrossCloudMenu = (content) => {
+  const crossCloudMenuEl = content.querySelector('.cross-cloud-menu');
+  if (!crossCloudMenuEl) return;
+
+  const crossCloudMenuContent = toFragment`<div class="feds-crossCloudMenu"></div>`;
+
+  [...crossCloudMenuEl.querySelector('ul').children].forEach((el, index) => {
+    const decoratedLink = decorateLinkGroup(el, index);
+    if (index === 0) decoratedLink.prepend(toFragment`${CONFIG.icons.home}`);
+    crossCloudMenuContent.append(toFragment`<div class="feds-crossCloudMenu-item">${decoratedLink}</div>`);
+  });
```

Comment:
Come to think of it, why not use `decorateElements({ elem: crossCloudMenuEl })` directly? We get a bit more markup, but we avoid swapping out entire elements for newly built ones